test/puppetlabs/puppetdb/config_test.clj

```diff
@@ -173,6 +169,15 @@
                      (.mkdir)
                      (.setWritable true))]
       (is (= (validate-vardir (vardir filename))
+             (vardir filename)))))
+
+  (testing "should support relative paths"
+    (let [filename "./target/totally/okay"]
+      (doto (fs/file filename)
+        (.deleteOnExit)
+        (.mkdirs)
```

Comment:
Perhaps, but in this case it's important that the path be relative, and at a glance it looks like that fn would return an absolute path.

In any case I'm more than happy to refactor this PR however the team feels is appropriate, assuming you guys are OK with the general spirit of the PR.